### ISSUE TITLE
AggRag app bugs

### ISSUE THREAD

**Issue by tarun-goyal** [genai-apps/aggrag#58]
@kangkan-fractal You can add the identified bugs here.

**Comment by kangkan-fractal** [comment]
- Importing a new cforge file my screen is stuck and I can't move (same raised by @lilit-fractal in Teams)

![Stuck and can't move](https://github.com/user-attachments/assets/8f494d9f-87ce-4cf3-a2fb-6e3452339055)

- Introductory Questions use case never loads for me. Also, the "usecas sample" usecase doesnt load. Plus "usecas" spellin wrong

![Image](https://github.com/user-attachments/assets/b08882e7-2e74-42db-8918-307f6c408a76)

- We need a few more simple and complex use-cases for people to start with

**Comment by tarun-goyal** [comment]
@santhosh-kumar12532 
- [x] Clicking on an iteration to open it, doesn't auto-close the menu

https://github.com/user-attachments/assets/3a8b1cbc-039b-43e0-8f83-2ec95874aa58

- [x] Disable "create index" until at least one file has been uploaded in the File Fields node.

**Comment by Munir-fractal** [comment]
Inconsistency with the "save" button styled similar to "export" and "import"

![Image](https://github.com/user-attachments/assets/2b25dcb1-e8ff-4534-8913-dcd5efa3f277)

**Comment by tarun-goyal** [comment]
@santhosh-kumar12532 Adding one more to the list.

- [x] When I add connectors between text/input data nodes and prompt node, and then "Save" an iteration. If I come back to this iteration later, these connectors disappear (I observed other changes are getting saved but not the connectors).

For some reason I'm not able to upload the screen recording here. We can connect once when you plan to pick this up.

**Comment by tarun-goyal** [comment]
@santhosh-kumar12532 Tested all the fixes:

Passed:
1. Importing a new cforge file my screen is stuck and I can't move it
2. Clicking on an iteration to open it, doesn't auto-close the menu
3. Introductory Questions use case never loads for me. Also, the "usecas sample" usecase doesnt load. Plus "usecas" spellin wrong --> **The iteration under introductory questions is blank so there is nothing to load - will need to remove that (I'm creating a new ticket to remove any redundant (default/sample) use-cases/iterations from AggRag). The typo has been fixed.**

These fixes are pending I guess because https://github.com/genai-apps/aggrag/pull/77 is yet to be merged. Will review once done.
4. Disable "create index" until at least one file has been uploaded in the File Fields node.
5. Inconsistency with the "save" button styled similar to "export" and "import"
6. When I add connectors between text/input data nodes and prompt node, and then "Save" an iteration. If I come back to this iteration later, these connectors disappear (I observed other changes are getting saved but not the connectors).
